util/dvsim/Deploy.py

```diff
@@ -437,7 +437,7 @@ def _set_attrs(self):
         self.full_name = self.sim_cfg.name + ":" + self.qual_name
         self.job_name += f"_{self.build_mode}"
         if self.sim_cfg.cov:
-            self.output_dirs += [self.cov_db_test_dir]
+            self.output_dirs += [self.cov_db_dir]
```

Comment:
More explanation here (only relevant for people who has access): [Google Doc](https://docs.google.com/document/d/1ic8Bhjj-vixzI17v5_9HoMRuGv-2cpqy2kkPdLdp3dI/edit?usp=sharing )

Reply:
This is slightly problematic. If the `cov_db_dir` from a test run contains updates to common files, then when the output is untar'ed locally, they will step on each other. Lets merge this in and see if we are actually running into any issue.

Reply:
Actually I thought about it again.. You are right, and this solution does not seem to work.. Because in our current method - we are collecting all coverage data into this `testdata` folder, then use `cov_merge` target to merge all the outputs.
Like you mentioned, if we run a nightly, new coverage will override the old ones, and we ended up only collect one coverage result.
Do you think it works if we output different coverage directories by tests, So under coverage folder, we will have `otp_ctrl_smoke_test1.vdb`, `otp_ctrl_smoke_test2.vdb`, etc, and we will merge all these `vdb`s.

Reply:
well that does not work either, build and run targets need to have the same `-cm_dir` coverage database. Otherwise there will be many duplicated files.

Reply:
Here's the link to what Staple does:
https://docs.google.com/presentation/d/1wxSmTLRd2xkWPgaRIHYMGKw4PCxJ7DMIbqcFmSe8wxo/edit#slide=id.g847b551bd3_0_45

Seems like the same idea as yours.

Reply:
Thank you Sri! Yeah I tried it in edacloud, actually looks like it is working with this implementation.